Core_Method/CoreMethod_FWD/SDIRK_FWD_Integrator.m

```diff
@@ -62,14 +24,30 @@
 function [ Tout, Yout, ISTATUS, RSTATUS, Ierr, stack_ptr, quadrature ] = SDIRK_FWD_Integrator( OdeFunction,...
     Tspan, Y, OPTIONS, Coefficient, adjStackFlag, adjQuadFlag, stack_ptr )
 
+
+    % Force initial value matrix to be N X 1.
+    if ( size(Y,2) == 1 )
+        % DO NOTHING
+    else
+        Y = transpose(Y);
+    end  
+
     % Get Problem Size
     NVAR = max(size(Y));
 
     Tinitial = Tspan(1);
     Tfinal = Tspan(2);
+    
+   if ( OPTIONS.storeCheckpoint == true )
+        Yout = zeros(NVAR,OPTIONS.Max_no_steps);
```

Comment:
Rename to Max_Num_Steps. With the dynamic reallocation of the solution trajectory do we need to do this?  #6